### PR TITLE
Large donation amounts for bouquet projects issue fixed

### LINKED ISSUE #152: Fix layout for displaying large donation amounts for Bouquet projects
**Describe the bug**
Large donation amounts do not fit in the layout for Bouquet projects, e.g. for Mexican currency MXN on Treesforjane.org at https://donate.plant-for-the-planet.org/?tenant=ten_1e5WejOp&to=proj_xkTKfkV2uKk9COvugnCpGl6c

**Is it on a Production or Development Build?**
production

**To Reproduce**
1. Go to https://donate.plant-for-the-planet.org/?tenant=ten_1e5WejOp&to=proj_xkTKfkV2uKk9COvugnCpGl6c
2. Choose currency MXN
3. See error

**Screenshots**
image

**Desktop (please complete the following information):**
- OS: latest MacOS
- Browser: latest Chrome

### PR DESCRIPTION
**Pull request by rishsingh07**
Fixes #152 
after
image

### CODE REVIEW

**Comment by norbertschuler** [comment]
- [ ] This happened when I opened https://donate-with-planet-7mk8d5dkf-planetapp.vercel.app/?tenant=ten_1e5WejOp&to=proj_xkTKfkV2uKk9COvugnCpGl6c

Bildschirmfoto 2021-11-16 um 13 51 36

**Comment by norbertschuler** [comment]
- [ ] Example for a country with currency with high amounts (Nigeria):

Bildschirmfoto 2021-11-16 um 13 54 51

**Review by norbertschuler — CHANGES_REQUESTED** [review]
Maybe we can further improve the UI? We should not omit the rounding and i18n styling formerly provided by the helper function. Maybe we can just tell the helper function to not display the currency symbol instead.

**Comment by sagararyal** [comment]
I agree, 
we shouldn't remove the currency formatting. eg, Euros in German should appear with ,. eg: 20,50 whereas in english 20.50

For the rounding, i think it is worth involving the backend team to ensure that cost isn't sent in multiple decimal points.

File: src/Donations/Micros/DonationTypes/BouquetDonations.tsx [hunk 1]
```diff
@@ -98,11 +98,7 @@ function FundingDonations({ setopenCurrencyModal }: Props): ReactElement {
                 </div> */}
                 <div className="funding-selection-option-text">
                   <span style={{ fontSize: "20px" }}>
-                    {getFormatedCurrency(
-                      i18n.language,
-                      currency,
-                      paymentSetup.unitCost * option.quantity
-                    )}
+                    {paymentSetup.unitCost * option.quantity}
```
Comment (norbertschuler) [inline src/Donations/Micros/DonationTypes/BouquetDonations.tsx:101]: I would propose to just do:
```
{getFormatedCurrency(
  i18n.language,
  '',
  paymentSetup.unitCost * option.quantity
)}
```
Bildschirmfoto 2021-11-17 um 13 39 23

File: src/Donations/Micros/DonationTypes/BouquetDonations.tsx [hunk 2]
```diff
@@ -98,11 +98,7 @@ function FundingDonations({ setopenCurrencyModal }: Props): ReactElement {
                 </div> */}
                 <div className="funding-selection-option-text">
                   <span style={{ fontSize: "20px" }}>
```
Comment (norbertschuler) [inline src/Donations/Micros/DonationTypes/BouquetDonations.tsx:100]: And to be able to show larger amount, just use a smaller font size, e.g. 16px?

**Review by norbertschuler — DISMISSED** [review]

File: src/Donations/Micros/DonationTypes/BouquetDonations.tsx (hunk 2 above)
Comment (rishsingh07) [reply src/Donations/Micros/DonationTypes/BouquetDonations.tsx:100]: there can be a situation where in one screen, 6 donations are there two having biggere amounts and rest having smaller. So I don't think it wil llook nice.

**Review by norbertschuler — APPROVED** [review]
I am fine with this optimization just not showing the currency as it is already displayed below for the custom amount:

Bildschirmfoto 2021-11-19 um 17 53 29

**Pull request merged**